text-ui-test/EXPECTED.TXT

```diff
@@ -1,7 +1,70 @@
-Hello from
- ____        _        
-|  _ \ _   _| | _____ 
-| | | | | | | |/ / _ \
-| |_| | |_| |   <  __/
-|____/ \__,_|_|\_\___|
+________________________________________
+Hello! I'm Downy.
+How can I help?
+________________________________________
 
+________________________________________
+There are no tasks!
+________________________________________
+
+________________________________________
+Okay! Added this task:
+  [T] [ ] borrow book
+Now you have 1 tasks in this list
+________________________________________
+
+________________________________________
+Okay! Added this task:
+  [D] [ ] return book (by: Sunday)
+Now you have 2 tasks in this list
+________________________________________
+
+________________________________________
+Okay! Added this task:
+  [E] [ ] meet friends (from: Mon 2pm to: Mon 4pm)
+Now you have 3 tasks in this list
+________________________________________
+
+________________________________________
+Here are the tasks in your list:
+1. [T] [ ] borrow book
+2. [D] [ ] return book (by: Sunday)
+3. [E] [ ] meet friends (from: Mon 2pm to: Mon 4pm)
+________________________________________
+
+________________________________________
+Nice! You've completed this task:
+  [T] [X] borrow book
+________________________________________
+
+________________________________________
+Nice! You've completed this task:
+  [D] [X] return book (by: Sunday)
+________________________________________
+
+________________________________________
+Ok! This task is not complete:
+  [D] [ ] return book (by: Sunday)
+________________________________________
+
+________________________________________
+Here are the tasks in your list:
+1. [T] [X] borrow book
+2. [D] [ ] return book (by: Sunday)
+3. [E] [ ] meet friends (from: Mon 2pm to: Mon 4pm)
+________________________________________
+
+________________________________________
+exceptions.InvalidCommandException: Invalid command entered
+Here are a list of valid commands:
+ - list
```

Comment:
I like how you show the list of valid commands when a user has entered an invalid command